frontend/src/lib/Quiver.ts

```diff
@@ -111,7 +112,110 @@ type IntervalClosed = "left" | "right" | "both" | "neither"
 
 // Our current Typescript version (3.9.5) doesn't support template literal types,
 // so we have to use string literals for now.
-type IntervalIndex = string
+type IntervalType = string
```

Comment:
nit: What was the reason why this got renamed? Was it before only something supported for index columns?

Reply:
We don't have any behavior change, but the previous name was not very precise as we have the same code to render columns(index) as well as values.  Now I want to fix it so that it doesn't suggest that it is only used for columns(index).